pkgs/development/libraries/openbabel2/default.nix

```
@@ -0,0 +1,31 @@
+{stdenv, fetchurl, fetchpatch, cmake, zlib, libxml2, eigen, python, cairo, pcre, pkgconfig }:
```

Because this is just a different version of the same library the path should be 

``pkgs/development/libraries/openbabel/2.nix``

Please correct me if this is not true.

```suggestion
{stdenv, lib, fetchurl, fetchpatch, cmake, zlib, libxml2, eigen, python, cairo, pcre, pkg-config }:
```